apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -118,59 +118,70 @@ export const pageRouter = router({
         .select("parentId")
         .executeTakeFirstOrThrow()
 
-      const pages = await db
-        .selectFrom("Resource")
-        // NOTE: CANNOT be `inner join`
-        // because we have have resources that are
-        // never published
-        .leftJoin(
-          "Version",
-          "Version.resourceId",
-          "Resource.publishedVersionId",
-        )
-        .innerJoin("Blob", (join) =>
-          join.on((eb) =>
-            // Step 1: If the `draftBlobId` is present on the `Resource`
-            // we should always use this because it's the latest one
-            eb.or([
-              eb("Blob.id", "=", eb.ref("Resource.draftBlobId")),
-              // Step 2: Otherwise, select the latest blob
-              // that was published
-              eb(
-                "Blob.id",
-                "in",
-                eb
-                  .selectFrom("Blob")
-                  .select("Blob.id")
-                  .leftJoin("Version", "Version.blobId", "Blob.id")
-                  .where("Version.blobId", "=", eb.ref("Blob.id"))
-                  .orderBy("publishedAt desc")
-                  .limit(1),
+      const result = await db
```

Comment:
Is this to get the unique categories of a collection?

Would something like this suffice?

```
SELECT DISTINCT 
    COALESCE(
        b.content->'page'->>'category', -- prefer draftBlob category if available
        vb.content->'page'->>'category' -- if not, we fallback to publishedVersion category
    ) AS category
FROM "Resource" r
LEFT JOIN "Blob" b ON r."draftBlobId" = b.id
LEFT JOIN "Version" v ON r."publishedVersionId" = v.id
LEFT JOIN "Blob" vb ON v."blobId" = vb.id
WHERE r.type = 'CollectionPage' AND r."siteId" = 36;

```